draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1065,6 +1065,7 @@ Error codes:
 | unknown anchor  | The last certificate in the chain both is not, and is not certified by, an accepted trust anchor. |
 | bad chain       | The alleged chain is not actually a chain of certificates.                                        |
 | bad certificate | One or more certificates in the chain are not valid (e.g., not properly encoded).                 |
+| try later       | The log is operational but is temporarily unable to accept new submissions.                       |
```

Comment:
Does this response overlap with 503 - Service Unavailable? It seems like using that (plus a Retry-After header) could indicate this as well.

https://tools.ietf.org/html/rfc7231#section-6.6.4